taxi/tests.py

```diff
@@ -1,3 +1,27 @@
 from django.test import TestCase
+from django.contrib.auth import get_user_model
+from taxi.models import Manufacturer, Car
 
-# Create your tests here.
+
+class ModelsTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="BMW", country="Germany")
+        self.assertEqual(str(manufacturer), f"{manufacturer.name} {manufacturer.country}")
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="test",
+            password="test123",
```

Comment:
The password provided here is stored as plain text, which is not secure. Consider using the `set_password` method to hash the password before saving the user.